ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -491,83 +385,59 @@ const PanelMain = fp.flow(
       ).includes(status as RuntimeStatus) &&
       runtimeCanBeCreated;
 
-    const renderUpdateButton = () => {
-      return (
-        <Button
-          aria-label='Update'
-          disabled={!runtimeCanBeUpdated}
-          onClick={() => {
-            requestAnalysisConfig(analysisConfig);
-            onClose();
-          }}
-        >
-          {updateMessaging.applyAction}
-        </Button>
-      );
-    };
+    const renderCreateOrUpdateButton = (
```

Comment:
alt: remove these render functions entirely and specify the components inline

Reply:
Do you mean to say that these would work better as proper components?

Reply:
Yes, that's a better way to word it